### PR TITLE
redis 7 sentinel support #754 

### PR DESCRIPTION
**Pull request by axeltomasson**

### CODE REVIEW

File: src/Connection/Aggregate/SentinelReplication.php [hunk 1]
```diff
@@ -381,7 +381,7 @@ protected function querySentinelForSlaves(NodeConnectionInterface $sentinel, $se
         $slaves = array();
 
         $payload = $sentinel->executeCommand(
-            RawCommand::create('SENTINEL', 'slaves', $service)
+            RawCommand::create('SENTINEL', 'replicas', $service)
```
Comment (tillkruss) [inline src/Connection/Aggregate/SentinelReplication.php:384]: This will break compatibility with older versions.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: src/Connection/Aggregate/SentinelReplication.php (hunk 1 above)
Comment (axeltomasson) [reply src/Connection/Aggregate/SentinelReplication.php:384]: Yeah sorry of course did not think of that.  Was fixated on that everything older then 7.0.x, 6.0.x and 6.2.x was EOL.

Comment (tillkruss) [reply src/Connection/Aggregate/SentinelReplication.php:384]: I wish 🙃

Comment (axeltomasson) [reply src/Connection/Aggregate/SentinelReplication.php:384]: Misuse profile to signal 7.0.0 version from config time?

~~~
        if (version_compare($sentinel->getParameters()->profile, '7.0') >= 0) {
            $payload = $sentinel->executeCommand(
                RawCommand::create('SENTINEL', 'replicas', $service)
            );
        } else {
            $payload = $sentinel->executeCommand(
                RawCommand::create('SENTINEL', 'slaves', $service)
            );
        }
~~~

Comment (rhl-jfm) [reply src/Connection/Aggregate/SentinelReplication.php:384]: > Yeah sorry of course did not think of that. Was fixated on that everything older then 7.0.x, 6.0.x and 6.2.x was EOL.

From a brief review of the Redis sources, SENTINEL REPLICAS is supported from Redis 5.0.

**Pull request closed without merging**